Types/Table.php

```diff
@@ -62,4 +65,40 @@ public static function alias(array $tokens) {
         if ($operation === SqlOperations::UPDATE) return $tokens['UPDATE'][0]['alias']['name'];
         return $tokens['FROM'][0]['alias']['name'];
     }
+
+    /**
+     * replaces the table in the tokens array with the given table
+     *
+     * @param  array $tokens
+     * @param  array $newTable
+     * @return array
+     */
+    public static function replace(array $tokens, $newTable) {
+        Assertions::assertHashMap('tokens', $tokens);
+
+        $operation = SqlOperation::create($tokens);
+        if ($operation === SqlOperations::INSERT) $tokens['INSERT'][1]['no_quotes']['parts'][0] = $newTable;
```

Comment:
Das klappt in dem fall nicht, weil tokens bei jeder Methode eine andere Struktur hat. Deshalb hatte ich hier ausnahmsweise mal sowas Böses gemacht